streaming/README.md

```diff
@@ -316,6 +325,65 @@ With the introduction of TLS/SSL, the parent-child communication behaves as show
 | Yes|-/force/optional|Yes|no|The parent-child stream is encrypted, provided that the parent has a valid TLS/SSL certificate. Otherwise, the child refuses to connect.|
 | Yes|-/force/optional|Yes|yes|The parent-child stream is encrypted.|
 
+### Streaming compression
+Streaming compression is supported in Netdata streaming protocol version 5+.
+#### OS dependencies
+* Streaming compression is based on [lz4 v1.9.0+](https://github.com/lz4/lz4). The [lz4 v1.9.0+](https://github.com/lz4/lz4) library must be installed in your OS in order to enable streaming compression. Any lower version will disable Netdata streaming compression for compatibility purposes between the older versions of Netdata agents.
+
+You can check if your netdata agent supports and was build with stream compression through the Netdata agent web api (browser/terminal).
+```
+curl -X GET http://localhost:19999/api/v1/info | grep 'Stream Compression'
+```
+
+OUTPUT:
+<pre>
+"buildinfo": "dbengine|Native HTTPS|Netdata Cloud|ACLK Next Generation|New Cloud Protocol Support|ACLK Legacy|TLS Host Verification|Machine Learning|<b>Stream Compression</b>|protobuf|JSON-C|libcrypto|libm|LWS v3.2.2|mosquitto|zlib|apps|cgroup Network Tracking|EBPF|perf|slabinfo",
+</pre>
+Note: If your OS doesn't support Netdata compression the `buildinfo` will not contain the `Stream Compression` statement.
+
+You can check if your netdata agent has stream compression enabled or disabled.
```

Comment:
Considering this sentence, should we add an information that `compression` is present only for netdata newer than `1.3x`?

Reply:
Yes you are right! I believe the right place to put this information is just right in the beginning of the streaming compression section.

Reply:
I agree, thanks!